modules/raycast/config.py

```diff
@@ -0,0 +1,6 @@
+def can_build(env, platform):
+    return env["tools"] and platform not in ["android", "ios"]
```

Comment:
If we decide not to build it on 32-bit for https://github.com/godotengine/godot/pull/44628#issuecomment-752416916, maybe this test should be further improved to match the oidn one. I'm not sure it makes sense to build embree for JavaScript tools?